### ISSUE TITLE
A reactive episode w/ RxJs

### ISSUE THREAD

**Issue by PipelineBaron** [nodeup/contribute#1]
https://github.com/Reactive-Extensions/RxJS

What is reactive?
Why reactive instead of streams?
Use cases.

https://twitter.com/mattpodwysocki << Maintainer
https://twitter.com/substack << Substack would be cool for feedback on VS streams

**Comment by dshaw** [comment]
Nice! @mattpodwysocki is awesome. There are several way we could go with this.

**Comment by mhart** [comment]
Yeah definitely - I've seen a lot of stream alternatives popping up on GH - perhaps too many are in their infancy (RxJS aside), but I'd love to hear about them. A lot of the newer ones seemed to grow out of issues with Streams2 as well.

Things like https://github.com/dominictarr/pull-stream from @dominictarr and https://github.com/creationix/min-stream from @creationix

**Comment by dshaw** [comment]
@mhart Oooh, nice. Alt Streams sounds like fun. Let's create a new show for that too. Probably with @isaacs too.

**Comment by dominictarr** [comment]
pull-streams and min streams are very similar, actually.

Would also be worth covering https://github.com/gozala/reducers so maybe
we'd want to get gozala in on such a show.

**Comment by mattpodwysocki** [comment]
I'd love to talk streams and FRP in addition to RxJS. Could be a fun show!

**Comment by dominictarr** [comment]
yeah, that would be a good angle: Streams meets FRP

**Comment by creationix** [comment]
I think I've finally stabilized my min-stream + continuable style APIs, so
if we talk about that, it shouldn't change the next week like it's been
doing the last few months.

Lots of FRP stuff, but with node style back-pressure and functions
everywhere!

**Comment by DamonOehlman** [comment]
Sounds good guys - definitely looking forward to this episode.  Is it worth getting @isaacs in on the action too, as I guess I'm interested in a constructive discussion as to where Streams2 is falling down.

**Comment by uris77** [comment]
+1

**Comment by mattpodwysocki** [comment]
I'd love to have @jhusain along for the ride as well since he uses RxJS at Netflix and can talk about how they use it in detail

**Comment by jhusain** [comment]
I'd love to paticipate.

**Comment by joyrexus** [comment]
+1.

Apropos @mattpodwysocki quote from the [JSJ podcast FRP episode](http://javascriptjabber.com/083-jsj-frp-and-rxjs-with-matthew-podwysocki/):

> Node streams, we [RxJS devs] definitely want to go there a lot more. Although there’s definitely a mismatch between what they do and what we do which could be a whole another show, believe me.

Also note the number of modules now on npm tagged [reactive](https://www.npmjs.org/browse/keyword/reactive), [rx](https://www.npmjs.org/browse/keyword/rx), or [frp](https://www.npmjs.org/browse/keyword/frp).
